gradle/runBootMicroserviceAcceptanceTests.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -x
+
+function dumpCurrentMicroInfraSpringVersionToFile {
+
+    ./gradlew -q currentVersion | grep "Project version" | sed s/Project\ version\:\ //g > ~/.microInfraSpringCurrentVersion.txt
+    cat ~/.microInfraSpringCurrentVersion.txt
+}
+
+function cloneAndDoBuild {
+
+    echo Checking out and building $1 with micro-infra-spring `cat ~/.microInfraSpringCurrentVersion.txt`
+    git clone https://github.com/4finance/"$1".git
+    cd "$1"
+    echo "microInfraSpringVersion="`cat ~/.microInfraSpringCurrentVersion.txt` >> gradle.properties
```

Comment:
Can't you just replace `micro-infra-spring` version to `+`? Also you can use `git patch` or something to avoid `sed` (if easier).